clippy_lints/src/derive.rs

```diff
@@ -176,6 +210,7 @@ impl<'tcx> LateLintPass<'tcx> for Derive {
 
             check_hash_peq(cx, item.span, trait_ref, ty, is_automatically_derived);
             check_ord_partial_ord(cx, item.span, trait_ref, ty, is_automatically_derived);
+            check_partial_eq_without_eq(cx, item.span, trait_ref, ty, is_automatically_derived);
```

Comment:
Can this be moved to `if is_automatically_derived` block and `is_automatically_derived` be removed from args?

Reply:
Thanks for catching that, will fix